cypress/integration/articleFlows/postSidebarActions.spec.js

```diff
@@ -1,4 +1,10 @@
 describe('Post sidebar actions', () => {
+  const runtimeStub = {
+    onBeforeLoad: (win) => {
+      Object.defineProperty(win.navigator, 'share', { value: true });
+    },
+  };
+
```

Comment:
it looks so simple but I know it's not 😂  I'm glad we have this in place now and can replicate where needed! 🎉